src/main/resources/io/viash/helpers/bashutils/ViashDockerFuns.sh

```diff
@@ -13,9 +13,9 @@ function ViashDockerInstallationCheck {
   fi
 
   ViashDebug "Checking whether the Docker daemon is running"
-  save=$-; set +e
+  local save=$-; set +e
   docker_version=$(docker version --format '{{.Client.APIVersion}}' 2> /dev/null)
```

Comment:
```suggestion
  local docker_version=$(docker version --format '{{.Client.APIVersion}}' 2> /dev/null)

Reply:
ahh right! I spotted that before and then lost track of it. `docker_version` isn't used, so first I thought it must be used somewhere else, but it turns out it's not used at all